src/control/server/ctl_storage.go

```diff
@@ -248,6 +248,8 @@ func (cs *ControlService) scanAssignedBdevs(ctx context.Context, statsReq bool)
 			if err != nil {
 				return nil, errors.Wrap(err, "create controller map")
 			}
+			cs.log.Debugf("instance %d: nvme ctrlr map for tier %d: %+v", ei.Index(),
```

Comment:
Is this debug necessary to leave in? The debug output has been growing a lot lately, and we need to be sure that the statements we're leaving in really add value.

Reply:
This debug helps me evaluate the reason for failures, @johannlombardi commented that it was a shame we didn't have enough information in the debug logs to understand why a given failure was happening so I'm trying to add relevant information to help us derive certain failure cases. I understand that this is a delicate balance, I'm just trying to satisfy requests.

Reply:
removed